### ISSUE TITLE
[APP SUBMITTED]: 'ascii' codec can't encode character u'\xf4' in position 44: ordinal not in range(128)

### ISSUE THREAD

**Issue by OmgImAlexis** [SiCKRAGETV/sickrage-issues#1815]
### INFO
Python Version: **2.7.6 (default, Mar 22 2014, 22:59:56) [GCC 4.8.2]**
Operating System: **Linux-3.13.0-48-generic-x86_64-with-Ubuntu-14.04-trusty**
Locale: unknown
Branch: **master**
Commit: SiCKRAGETV/SickRage@e6c4cfd6546b74d5527361f6f0bdc3adc232bdb0
Link to Log: https://gist.github.com/61b462b527a108f48c23
### ERROR
```
POSTPROCESSER :: Exception generated in thread POSTPROCESSER: ascii : /mnt/downloads/DONE/tvshow/Hannibal.S01E11.Rôti.720p.WEB-DL.DD5.1.H.264-NTb.mkv : error 44 : error 45 : ordinal not in range(128)
```
---
_STAFF NOTIFIED_: @SiCKRAGETV/owners @SiCKRAGETV/moderators

**Comment by neoatomic** [comment]
Problem is the special character in "Rôti"

**Comment by fernandog** [comment]
https://sickrage.tv/forums/forum/help-support/general-support/12825-ensuring-your-locale-s-are-set-properly-to-avoid-unicode-issues

**Comment by OmgImAlexis** [comment]
@fernandog both my ~/.profile and ~/.bashrc have the 3 lines in them, I ran ``source ~/.bashrc`` and then tried again which gave me the same issue. I restarted Sickrage and got the same issue. I even restarted my VM and I still get it. I made sure to check that the user is the correct one that's listed on the /config page. Anything else I can try? I know I could just rename it but I'd rather actually fix the issue.
